contents/julia_basics.md

```diff
@@ -2363,3 +2360,260 @@ sco(s; process=catch_show)
 
 > **_NOTE:_**
 > For more complex HTTP interactions such as interacting with web APIs, see the [`HTTP.jl` package](https://github.com/JuliaWeb/HTTP.jl).
+
+### Project Management {#sec:project_management}
+
+One last thing from Julia's standard library for us to cover is the **`Pkg` module**.
+As described in @sec:programmers,
+Julia offers a **native project management solution**,
+with dependencies and version control tightly controlled, manageable, and replicable.
+
+Unlike traditional package managers,
+which install and manage a single global set of packages,
+Julia's package manager is designed around "environments":
```

Comment:
```suggestion
Unlike traditional package managers,
which install and manage a single global set of packages,
Julia's package manager is designed around "environments" like Python's `venv`:
```

Reply:
I think that this is not the intention here.
`venv` is very different from Julia's `Pkg.jl`.

One major difference is the centralized package versioning local repository.
If you have 3 project that all of them uses `DataFrames.jl` version 1.4 then you'll have only one copy/precompilation of it in your `.julia/` folder.
Whereas in `venv` you'll have 3 local dot folder each one having its own copy of the packages.